package.json

```diff
@@ -24,37 +26,29 @@
   },
   "homepage": "https://github.com/readmeio/fetch-har#readme",
   "dependencies": {
-    "parse-data-url": "^4.0.1"
+    "formdata-node": "^4.3.2",
```

Comment:
Should this be an optionalDependency so we don't send it into a browser environment? Is that possible to conditionally bring it in?